chcemvediet/apps/obligees/tests/test_forms.py

```diff
@@ -115,102 +121,106 @@ def test_submitted_with_nonexisting_obligee_name(self):
         self.assertInHTML(u'<label for="id_obligee">Obligee:</label>', rendered)
         self.assertInHTML(u"""
-                <input class="autocomplete" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text">
+                <input class="form-control pln-autocomplete" data-autocomplete-url="{url}" name="obligee" data-name="obligee" type="text" value="">
                 """.format(url=reverse(u'obligees:autocomplete')), rendered)
         self.assertInHTML(u"""
-                <div class="obligee_widget_details obligee_widget_hide">
-                  <span class="obligee_widget_street"></span><br>
-                  <span class="obligee_widget_zip"></span> <span class="obligee_widget_city"></span><br>
-                  E-mail: <span class="obligee_widget_email"></span>
+                <div class="chv-obligee-widget-details chv-obligee-widget-hide">
+                  <span class="chv-obligee-widget-street"></span><br>
+                  <span class="chv-obligee-widget-zip"></span> <span class="chv-obligee-widget-city"></span><br>
+                  {}: <span class="chv-obligee-widget-email"></span>
+                  <span class="chv-obligee-widget-no-email">
+                  {}
+                    <span class="pln-with-tooltip" data-toggle="tooltip" data-placement="right" title="{}">
+                      <span class="chv-icon-stack chv-icon-lg">
+                        <i class="chv-icon icon-circle chv-color-blue-ll"></i>
+                        <i class="chv-icon chv-icon-inv chv-icon-half icon-help"></i>
+                      </span>
+                    </span>
+                  </span>
                 </div>
-                """, rendered)
+                """.format(_(u'obligees:obligee_widget:email'), _(u'obligees:obligee_widget:no_email'), lorem), rendered)
 
-    @unittest.skip(u'FIXME')
     def test_new_form_with_custom_widget_class_and_attributes(self):
         form = self.FormWithWidgetAttrs()
         rendered = self._render(u'{{ form }}', form=form)
         self.assertInHTML(u"""
-                <input class="autocomplete custom-class" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text" custom-attribute="value">
+                <input class="form-control pln-autocomplete custom-class" data-autocomplete-url="{url}" name="obligee" data-name="obligee" type="text" value="" custom-attribute="value">
                 """.format(url=reverse(u'obligees:autocomplete')), rendered)
 
-    @unittest.skip(u'FIXME')
     def test_new_form_with_initial_value_as_obligee_instance(self):
         names = [u'aaa', u'bbb', u'ccc', u'ddd']
-        oblgs = [self._create_obligee(name=n, street=u'%s street' % n, city=u'%s city' % n, zip=u'12345', emails=u'%s@a.com' % n) for n in names]
+        oblgs = [self._create_obligee(name=n, street=u'{} street'.format(n), city=u'{} city'.format(n), zip=u'12345', emails=u'{}@a.com'.format(n)) for n in names]
```

Comment:
Aj na inych v miestach v kode som si vsimol ze nahradzas formatovanie strigov z `%` na `.format`. Tak postupne budem nahradzat aj na miestach kde to este nie je.